api/src/main/java/org/openmrs/module/eptsreports/reporting/library/queries/TxMlQueries.java

```diff
@@ -60,7 +60,7 @@ public static String getPatientsWhoMissedAppointment(
             + "			p.voided = 0 AND e.voided = 0 AND o.voided = 0"
             + "			AND e.encounter_type = %d "
             + "			AND o.concept_id = %d"
-            + "			AND e.encounter_datetime <=:endDate AND e.location_id=:location "
+            + "			AND o.value_datetime <=:endDate AND e.location_id=:location "
```

Comment:
Is this change intended?

Reply:
yes according with the spec it is

Reply:
Please update pull request title with the ticket reference.

Reply:
Is this related to any ticket?